src/components/utils/onfidoApi.ts

```diff
@@ -128,6 +123,30 @@ export const uploadDocument = (
   })
 }
 
+export const uploadPoADocument = (
+  payload: UploadDocumentPayload,
+  url: string | undefined,
+  token: string | undefined,
+  onSuccess?: SuccessCallback<DocumentImageResponse>,
+  onError?: ErrorCallback
+): Promise<DocumentImageResponse> => {
+  const endpoint = `${url}/v3.3/documents`
+
+  return uploadDocumentMain(payload, endpoint, token, onSuccess, onError)
+}
+
+export const uploadDocument = (
+  payload: UploadDocumentPayload,
+  url: string | undefined,
+  token: string | undefined,
+  onSuccess?: SuccessCallback<DocumentImageResponse>,
+  onError?: ErrorCallback
+): Promise<DocumentImageResponse> => {
+  const endpoint = `${url}/v3/documents`
+
+  return uploadDocumentMain(payload, endpoint, token, onSuccess, onError)
+}
```

Comment:
There is no need to separate. You can update to v3.3 or keep v3 (there are no changes in the doc upload endpoint).

Reply:
I think I've been able to update it to v3.3

Reply:
What I meant is you don't need to separate. You could've just updated the existing `uploadDocument` to v3.3 instead of having to routes.